// atom-notify/dist/index.js
(function() {
  module.exports = function(title) {
    if (title == null) {
      title = '';
    }
    return {
      addInfo: function(message, arg) {
        var dismissable;
        dismissable = (arg != null ? arg : {}).dismissable;
        return atom.notifications.addInfo(title, {
          detail: message,
          dismissable: dismissable
        });
      },
      addSuccess: function(message, arg) {
        var dismissable;
        dismissable = (arg != null ? arg : {}).dismissable;
        return atom.notifications.addSuccess(title, {
          detail: message,
          dismissable: dismissable
        });
      },
      addError: function(message, arg) {
        var dismissable;
        dismissable = (arg != null ? arg : {}).dismissable;
        return atom.notifications.addError(title, {
          detail: message,
          dismissable: dismissable
        });
      },
      addWarning: function(message, arg) {
        var dismissable;
        dismissable = (arg != null ? arg : {}).dismissable;
        return atom.notifications.addWarning(title, {
          detail: message,
          dismissable: dismissable
        });
      }
    };
  };

}).call(this);
